src/ec/packages/ec-component-site-header/site-header.html.twig

```diff
@@ -117,24 +112,24 @@
   <div class="ecl-site-header__container ecl-container">
     <div class="ecl-site-header__banner">
       <a 
-        class="ecl-link ecl-link--standalone" 
-        href="{{ _header_link.url }}" 
-        aria-label="{{ _header_link.aria_label }}"
+        class="ecl-link ecl-link--standalone"
+        href="{{ _logo.url }}"
+        aria-label="{{ _logo.title }}"
       >
         <img 
-          alt="{{ _header_image.alt }}" 
-          title="{{ _header_image.title }}" 
-          class="ecl-site-header__logo-image" 
-          src="{{ _header_image.src }}"
+          alt="{{ _logo.alt }}"
+          title="{{ _logo.title }}"
+          class="ecl-site-header__logo-image"
+          src="{{ _logo.src }}"
         />
       </a>
       <div class="ecl-site-header__selector">
         <a 
-          class="ecl-link ecl-link--standalone" 
-          href="{{ _language.url }}"
-          data-ecl-language-selector
+          class="ecl-link ecl-link--standalone ecl-site-header__selector-link"
+          href="{{ _language_selector.url }}"
+          data-ecl-language-selector="true"
         >
-          {{- _language.label -}}
+          {{- _language_selector.name -}}
           <span class="ecl-site-header__language-icon">
             {% include '../ec-component-icon/icon.html.twig' with {
```

Comment:
I see this attr in ECL for icons in the site-header `data-ecl-icon="true"`, not really sure it has any relevance, though.

Reply:
ok, apparently this is not relevant.